king.yml

```diff
@@ -36,11 +37,11 @@
 #    - common-nodejs
 #    # User configs and stuff
 #    - mysql_configs
-#    - nginx_configs
+#    - nginx_sites_king
```

Comment:
This should be avoided in the future.

But good enough for know, this role should be split into multiple other roles anyway...